web/src/lib/domain-logic/getNavBarBusinessTitle.ts

```diff
@@ -26,6 +34,15 @@ export const getNavBarBusinessTitle = (
     }
   };
 
+  if (foreignBusinessType === "NEXUS") {
+    if (needsNexusDbaName && nexusDbaName === "") {
+      return Config.navigationDefaults.navBarUnnamedDbaBusinessText;
+    }
+    if (needsNexusDbaName && nexusDbaName !== "") {
+      return nexusDbaName;
+    }
+  }
```

Comment:
Looking at the rest of this file it seems like there might be a way to better structure this, but I'm not totally sure what that looks like even after a bit of thought. Advice is appreciated.

Reply:
I think this new code should go in the "determineName" function.

The overall flow of logic is "if they have entered a name, use that (via some logic to figure out what to use). Otherwise use a string via some other logic"

This new code falls under the "they have entered a name, use that via some logic" bucket and should therefore go in `determineName`

Reply:
Yeah, that makes sense, I was thinking it might be cool to somehow get this next the other "Foreign" user code but I don't think its worth it